src/token/erc721/erc721.cairo

```diff
@@ -337,55 +358,74 @@ mod ERC721Component {
             src5_component.register_interface(interface::IERC721_METADATA_ID);
         }
 
+        /// Returns the owner address of `token_id`.
+        fn _owner_of(self: @ComponentState<TContractState>, token_id: u256) -> ContractAddress {
+            self.ERC721_owners.read(token_id)
+        }
```

Comment:
The public method still panics, this is an internal one, I don't feel like the warning is necessary. Maybe a changelog-specific entry, but we would need to add a few, and since we are breaking that often, I think we don't need to be that specific with the internal API.

Reply:
> The public method still panics, this is an internal one, I don't feel like the warning is necessary.

My concern is the internal fn panicked in this lib for so long, but yeah. It _is_ internal and we have everything documented in the API doc so that should be enough

> Maybe a changelog-specific entry, but we would need to add a few, and since we are breaking that often, I think we don't need to be that specific with the internal API.

Agreed, it's probably not necessary to be that specific especially in this case